source/index.html

```diff
@@ -126,89 +137,112 @@ <h2 class="visually-hidden>">Тарифы и доступные опции.</h2>
+              </th>
+              <th class="price-table__option price-table__option--yes">
+                <span class="visually-hidden">Доступная опция</span>
+              </th>
+              <th class="price-table__option price-table__option--yes">
+                <span class="visually-hidden">Доступная опция</span>
+              </th>
             </tr>
             <tr>
               <th class="price-table__option">Смайлики</th>
-              <th class="price-table__option price-table__option--no"></th>
-              <th class="price-table__option price-table__option--yes"></th>
-              <th class="price-table__option price-table__option--yes"></th>
+              <th class="price-table__option price-table__option--no">
+                <span class="visually-hidden">Недоступная опция</span>
+              </th>
+              <th class="price-table__option price-table__option--yes">
+                <span class="visually-hidden">Доступная опция</span>
+              </th>
+              <th class="price-table__option price-table__option--yes">
+                <span class="visually-hidden">Доступная опция</span>
+              </th>
             </tr>
             <tr>
               <th class="price-table__option">Комментарии</th>
-              <th class="price-table__option price-table__option--no"></th>
-              <th class="price-table__option price-table__option--no"></th>
-              <th class="price-table__option price-table__option--yes"></th>
+              <th class="price-table__option price-table__option--no">
+                <span class="visually-hidden">Недоступная опция</span>
+              </th>
+              <th class="price-table__option price-table__option--no">
+                <span class="visually-hidden">Недоступная опция</span>
+              </th>
+              <th class="price-table__option price-table__option--yes">
+                <span class="visually-hidden">Доступная опция</span>
+              </th>
             </tr>
           </tbody>
         </table>
-        <div class="price-table__pagination pagination">
-          <ul class="pagination__list pagination-list">
-            <li class="pagination-list__item"></li>
-            <li class="pagination-list__item pagination-list__item--current"></li>
-            <li class="pagination-list__item"></li>
-          </ul>
+        <div class="price__pagination slider-pagination">
+            <button class="slider-pagination__toggle" type="button">
+              <span class="visually-hidden">Переключить слайд.</span>
+            </button>
+            <button class="slider-pagination__toggle slider-pagination__toggle--current" type="button">
+              <span class="visually-hidden">Текущий слайд.</span>
+            </button>
+            <button class="slider-pagination__toggle" type="button">
+              <span class="visually-hidden">Переключить слайд.</span>
+            </button>
```

Comment:
Чего-то с вложенностью только случилось
```suggestion
        <div class="price__pagination slider-pagination">
          <button class="slider-pagination__toggle" type="button">
            <span class="visually-hidden">Переключить слайд.</span>
          </button>
          <button class="slider-pagination__toggle slider-pagination__toggle--current" type="button">
            <span class="visually-hidden">Текущий слайд.</span>
          </button>
          <button class="slider-pagination__toggle" type="button">
            <span class="visually-hidden">Переключить слайд.</span>
          </button>
```